config/hathitrust_config.rb

```diff
@@ -11,33 +11,26 @@ def volume_writer
           subsets.map do |subset|
             [subset, subset_volume_writer(subset)]
           end
-          .concat([superset, superset_volume_writer])
+          .concat([[superset, superset_volume_writer]])
           .to_h
       end
 
       def src_path_resolver
         @src_path_resolver ||=
-          src_parent_dir.map do |profile, dir|
-            [profile, PairtreePathResolver.new(Pathname.new(dir))]
-          end
-          .to_h
+          PairtreePathResolver.new(Pathname.new(src_parent_dir))
       end
 
       def volume_repo
         @volume_repo ||=
           subsets.map do |subset|
             [subset, subset_volume_repo]
           end
-          .concat([superset, superset_volume_repo])
+          .concat([[superset, superset_volume_repo]])
```

Comment:
This is basically appending [:full, "Rights+Feed"] to the array of arrays.  When I replace the volume repos with strings, the result looks like this:
```
superset
=> :full
superset_volume_repo
=> "Rights+Feed"
subset_volume_repo
=> "Rights"
volume_repo             
=> [[:pd, "Rights"], [:pd_open, "Rights"], [:pd_world, "Rights"], [:pd_world_open, "Rights"]]
volume_repo.concat([[superset, superset_volume_repo]])
=> [[:pd, "Rights"], [:pd_open, "Rights"], [:pd_world, "Rights"], [:pd_world_open, "Rights"], [:full, "Rights+Feed"]]
```
Seeing that all makes sense now.  I don't have a good handle on what about this is irking me, but something here blocked me from grok'ing it immediately.  Seems like the [set, set_volume_repo] are inexorably linked. 

Perhaps it's a naming issue.  I think of "volume repo" as a repository of volumes.  This is actually a mapping of the volume subset to the rights database?  or in the case of the full set, the rights and rights and feed backend database? 

 Refactor later I guess.